pages/common/logstash.md

```diff
@@ -0,0 +1,21 @@
+# logstash
+
+> An ETL tool, usually used to ingest data from various sources, like databases and flat log files into elasticsearch.
+
+- Boilerplate configuration for logstash to provide logic for extract, transform and load, respectively:
+
+`input {}`
+`filter {}`
+`output {}`
+
+- Check validity of a logstash configuration:
+
+`logstash --configtest --config {{logstash-config.conf}}`
+
+- Run logstash using configuration:
+
+`sudo logstash --config {{logstash-config.conf}}`
+
+- Run logstash with inline configuration string:
```

Comment:
Suggest changing it to "Run logstash with the most basic inline configuration string"